static/app/components/replays/replayBreadcrumbs.tsx

```diff
@@ -0,0 +1,50 @@
+import React, {useState} from 'react';
```

Comment:
I don't believe you need to import `React` here.

Do you know if something is auto-importing this for you?

Reply:
Ahh, `React` is here because of the `<React.Fragment>` lower down, which isn't doing anything